website/content/docs/architecture/catalog/v2.mdx

```diff
@@ -0,0 +1,74 @@
+
+The following table describes resources in the v2 catalog and compares them to the v1 catalog and Kubernetes resources. It also states whether these resources are defined by the user or computed by Consul when it registers a service.
+
+| v2 Catalog resource | Description | Catalog v1 resource analogue | Kubernetes resource analogue | Source |
+| :------------------ | :---------- | :--------------------------- | :--------------------------- | :----- |
+| Service | The name of the service Consul registers a workload under | Service | None  | Defined by user in Kubernetes |
+| Node | The address of the node where the workload runs. | Node | Node | Computed by Consul |
+| Workload | An application instance running in a set of one or more Pods scheduled according to a Kubernetes Workload resource such as a Deployment or StatefulSet. | Service instance | Kubernetes Workloads | Defined by user in Kubernetes |
+| Workload identities | Provides a distinct identity for a Kubernetes Workload to assume in a Kubernetes cluster. | Service instance | Kubernetes Service Accounts | Defin by user in Kubernetes |
+| Service endpoints | Maps services to workload addresses and endpoints. | None  | Service backend | Computed by Consul |
+| Health status | A resource for reporting the health status of a workload | Agent check | None | Computed by Consul |
+| Health check | A resource for defining the health checks for a workload. | Health check | Liveness, Readiness, and Startup Probes | Defined by user in Kubernetes |
+| Proxy configuration | Represents a configuration for a sidecar or gateway proxy. | `Proxy` field in service definition | None | Defined by user in Consul |
+| Destinations | Represents explicit service upstreams | None | Upstream Service annotation  | Defined by user in Kubernetes |
+| Traffic permissions | Enables L4 traffic authorization according to workload identity instead of service identity. | Service intentions | None | Defined by user in Consul |
+
+Refer to [`consul resource`](/consul/docs/k8s/multiport/reference/resource-command) for more information about using the Consul CLI to interact with the v2 catalog.
+
+## Changes to Consul’s existing architecture
+
+The change in data models introduced by the v2 Catalog API impacts several aspects of Consul’s operations.
+
+### Traffic permissions resource replaces service intentions
+
+The most significant change to Consul’s architecture and operations when using the v2 catalog structure is the introduction of the TrafficPermissions resource. This resource replaces the service intentions configuration entry, and enables authorized service-to-service communication for both L4 and L7 applications.
+
+For more information about this resource, including example configurations, refer to [TrafficPermissions configuration reference](/consul/docs/k8s/multiport/reference/trafficpermissions).
+
+### HTTPRoute and GRPCRoute resources for L7 traffic management
```

Comment:
Maybe we should say TCPRoute, HTTPRoute, and GRPCRoute resources are used for traffic management rather than making this an L7 specific section. I think we want to describe that all traffic management is controlled via these resources.